packages/server/src/auth/auth.service.ts

```diff
@@ -193,4 +209,28 @@ export class AuthService {
       return null;
     }
   }
+
+  /**
+   * Refresh access token using refresh token.
+   *
+   * @param refreshToken
+   * @returns New access token
+   */
+  async refreshAccessToken(refreshToken: string): Promise<AccessToken> {
+    try {
+      const decodedRefreshToken = this.jwtService.verify(refreshToken);
+      const userId = decodedRefreshToken['id'];
+      const projectId = decodedRefreshToken['projectId'];
+
+      const payload = { id: userId, projectId: projectId, type: 'access' };
+      const newAccessToken = this.jwtService.sign(payload, { expiresIn: process.env.JWT_EXPIRATION });
+
+      return {
+        accessToken: newAccessToken,
+        refreshToken: refreshToken
```

Comment:
You also want to generate a new refresh token, since that token will also expire. So as long as they are active they should be good. To long and they will need to sign in again.